src/components/Dashboard/PendingTxListItem/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement } from 'react'
```

Comment:
Can you please organize all the widget components in one folder?

```
src
--| Dashboard
----| PendingTxns
------ | TxRow.tsx
------ | TxList.tsx
```

Reply:
Move it to that structure